tests/integration/session/test_sign_out_and_exit.py

```diff
@@ -126,39 +86,3 @@ def test_redirects_to_account_service_log_out_url_when_present(self):
         )
         self.get(SIGN_OUT_URL_PATH)
         self.assertInUrl(ACCOUNT_SERVICE_LOG_OUT_URL_PATH)
-
-
-class TestExitPostSubmissionWithHubCensusTheme(TestExitPostSubmissionTestCase):
```

Comment:
This should be repurposed to test the logic:

```
    log_out_url = (
        cookie_session.get("account_service_log_out_url", url_for(".get_signed_out"))
        if cookie_session
        else get_survey_config().base_url
    )
```

Is that behaviour tested elsewhere?

Reply:
I've added tests to test_session_sign_out.py